docs/pages/guides/typescript.mdx

```diff
@@ -104,18 +104,6 @@ Some language features may require additional configuration. For example, if wan
 
 > Available in SDK 49 and higher.
 
-In SDK 49 projects, you'll need to enable path aliases in the project's [app config](/workflow/configuration/):
```

Comment:
Let's use a tabs component here like this https://docs.expo.dev/routing/installation/#setup-entry-point